lib/mutant/result.rb

```diff
@@ -269,6 +270,15 @@ def timeout?
         !isolation_result.timeout.nil?
       end
 
+      # Test for unexpected process abort
+      #
+      # @return [Boolean]
+      def process_abort?
+        process_status = isolation_result.process_status or return false
```

Comment:
There is a noop isolation (used for debugging) that does NOT create a process status, hence we have to have this branch.

I wish we had a type system to cover all this cases.